playbooks/hail_query_on_batch.md

```diff
@@ -0,0 +1,106 @@
+# Hail: Query on Batch Playbook
+
+This playbook will describe some common errors for Hail's Query on Batch, and how to debug them. If you can't find an answer here, please raise an issue on #software-questions, and then come back and update this manual!
```

Comment:
It's important to consider the concept of ownership for documentation more broadly. Whilst it would be very, very nice if documentation were maintained collaboratively, I am not sure that you can rely on the user to update your documentation. What governance roles does software have behind the scenes to ensure that users' experiences are captured reliably in the docs?

Reply:
Thanks for the prompt!

I see this PR as a seed for shared-documentation, and not just software-team owned. Although I can see the framing for analysts as users, we as the data team are more deeply ingrained than that. There's a strong collaborative relationship between our teams, and not just software providing a service.

I hope (and will push) that analysis teams will see the most benefit by maintaining this doc.